util/Migrator/Migrator.csproj

```diff
@@ -2,6 +2,7 @@
 
   <ItemGroup>
     <EmbeddedResource Include="DbScripts\**\*.sql" />
+    <EmbeddedResource Include="DbScripts_data_migration\**\*.sql" />
```

Comment:
❓ Is the `_data_migration` in docs somewhere, or a recommendation? `DbDataMigrationScripts` or `DbDataMigrations` is cleaner to me.

Reply:
We've got another card to rename this folder: https://bitwarden.atlassian.net/browse/DEVOPS-1520